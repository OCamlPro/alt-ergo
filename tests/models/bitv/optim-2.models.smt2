; Test primitive optimization
(set-logic ALL)
(set-option :produce-models true)
(declare-const x (_ BitVec 32))
(minimize (bvnot x))
(check-sat)
(get-model)
(get-objectives)
