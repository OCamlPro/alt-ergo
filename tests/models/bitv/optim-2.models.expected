
unknown
(
  (define-fun x () (_ BitVec 32) #xffffffff)
)
(objectives 
  ((bvnot x) #x00000000)
)